test/framework/interface.go

```diff
@@ -355,6 +360,18 @@ func WithCtlOpts(opts map[string]string) KumaDeploymentOption {
 	})
 }
 
+func WithCPDockerRunOptions(options []string) KumaDeploymentOption {
+	return KumaOptionFunc(func(o *kumaDeploymentOptions) {
+		o.dockerRunOptions = options
+	})
+}
+
+func WithCPDockerVolumes(volumes ...string) KumaDeploymentOption {
```

Comment:
Name `WithDockerVolumes` is taken in this package, so adding a `CP` into the name for the scenario of installing Kuma CP.